ghost/core/core/server/data/migrations/versions/5.57/2023-07-27-11-47-49-create-donation-events.js

```diff
@@ -0,0 +1,19 @@
+// For information on writing migrations, see https://www.notion.so/ghost/Database-migrations-eb5b78c435d741d2b34a582d57c24253
+
+const {addTable} = require('../../utils');
+
+module.exports = addTable('donation_payment_events', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    name: {type: 'string', maxlength: 191, nullable: true},
+    email: {type: 'string', maxlength: 191, nullable: false, unique: true},
```

Comment:
Curious about `unique: true` for the email field. Why should this be unique? An user with email "abc@example.com" should be able to do multiple donations, and each donation will create a new `donation_payment_event`, right?

Reply:
Good catch, this was definitely not intended - copy paste mistake 🙃 I'll remove it